src/vcpkg/base/cmd-parser.cpp

```diff
@@ -0,0 +1,82 @@
+#include <vcpkg/base/cmd-parser.h>
+#include <vcpkg/base/strings.h>
+
+#include <stdint.h>
+
+#include <algorithm>
+
+namespace
+{
+    using namespace vcpkg;
+
+    static void help_table_newline_indent(std::string& target)
+    {
+        target.push_back('\n');
+        target.append(34, ' ');
+    }
+
+    static constexpr ptrdiff_t S_MAX_LINE_LENGTH = 100;
+}
+
+namespace vcpkg
+{
+    void HelpTableFormatter::format(StringView col1, StringView col2)
+    {
+        // 2 space, 31 col1, 1 space, 65 col2 = 99
```

Comment:
maybe use a const variable  (`COL1_WIDTH = 31`, `COL2_START = 34`) instead of quasi-magic numbers?